tools/__tasks__/compile/css/sass.js

```diff
@@ -0,0 +1,94 @@
+const fs = require('fs');
+const path = require('path');
+
+const sass = require('node-sass');
+const mkdirp = require('mkdirp');
+const glob = require('glob');
+const postcss = require('postcss');
+const autoprefixer = require('autoprefixer');
+const pxtorem = require('postcss-pxtorem');
+const pify = require('pify');
+
+const {src, target} = require('../../config').paths;
+const sassDir = path.resolve(src, 'stylesheets');
+
+const sassDefaults = {
+    outputStyle: 'compressed',
+    sourceMap: true,
+    precision: 5
+};
+
+const browserslist = [
+    'Firefox >= 26',
+    'Explorer >= 10',
+    'Safari >= 5',
+    'Chrome >= 36',
+
+    'iOS >= 5',
+    'Android >= 2',
+    'BlackBerry >= 6',
+    'ExplorerMobile >= 7',
+
+    '> 2% in US',
+    '> 2% in AU',
+    '> 2% in GB'
+];
+
+const remifications = {
+    replace: true,
+    root_value: 16,
+    unit_precision: 5,
+    prop_white_list: []
+};
+
+
+const renderSass = pify(sass.render);
+const saveCSS = pify(fs.writeFile);
+const getFiles = query => glob.sync(path.resolve(sassDir, query));
+
+const compile = (query, {browsers, remify = true}) => Promise.all(
```

Comment:
Do you think we need to get some docs/typing in place in some of these? Do you have a plan for that?